### PR TITLE
Make spaceman horizontal... but left-ly!

### PR DESCRIPTION
**Pull request by DisturbHerb**
[QOL]

## About the PR 
Any atom capable of resting has a 50/50 chance of falling to the left or right. The code is sorta ugly.

## Why's this needed? 
https://user-images.githubusercontent.com/66253095/125020840-e7a8b980-e0bc-11eb-9ddb-d050710dd641.mp4

### CODE REVIEW

**Comment by DisturbHerb** [comment]
this can't possibly go wrong

**Comment by BatElite** [comment]
The start of a new era

File: code/WorkInProgress/AnimationLibrary.dm [hunk 1]
```diff
@@ -1001,9 +1001,12 @@ proc/muzzle_flash_any(var/atom/movable/A, var/firing_angle, var/muzzle_anim, var
 	if(!istype(A))
 		return
 	if(stand)
-		animate(A, pixel_x = 0, pixel_y = 0, transform = A.transform.Turn(-90), time = 3, easing = LINEAR_EASING, flags=ANIMATION_PARALLEL)
+		animate(A, pixel_x = 0, pixel_y = 0, transform = A.transform.Turn(A.rest_mult * -90), time = 3, easing = LINEAR_EASING, flags=ANIMATION_PARALLEL)
+		A.rest_mult = 0
 	else
-		animate(A, pixel_x = 0, pixel_y = -4, transform = A.transform.Turn(90), time = 2, easing = LINEAR_EASING, flags=ANIMATION_PARALLEL)
+		var/fall_left_or_right = pick(1,-1) //A multiplier of one makes the atom rotate to the right, negative makes them fall to the left.
```
Comment (ZeWaka) [inline code/WorkInProgress/AnimationLibrary.dm:1007]:
```suggestion
		var/fall_left_or_right = pick(1, -1) //A multiplier of one makes the atom rotate to the right, negative makes them fall to the left.
```

**Review by ZeWaka — APPROVED** [review]

**Comment by ZeWaka** [comment]
i click spaceman, they go horizontal. succes

**Review by MarkNais — COMMENTED** [review]
I'm not jazzed about an additional atom var. I also can't think of a *great* way to implement this easily.

**Pull request merged**